test/liveValidatorTests.ts

```diff
@@ -558,6 +558,24 @@ describe("Live Validator", () => {
         // console.dir(validationResult, { depth: null, colors: true })
       })
     })
+    it(`should match operation in case of extension resource`, async () => {
```

Comment:
could you add a test which can verify your change in "sway" ?

Reply:
This test is for rpaas scenario, I added one more for arm scenario.